TRLevelControl/IO/TRLevelReader.cs

```diff
@@ -89,4 +89,35 @@ public List<TRTexImage32> ReadImage32s(long numImages)
         }
         return images;
     }
+
+    public List<TRColour> ReadColours(long numColours)
+    {
+        List<TRColour> colours = new((int)numColours);
+        for (long i = 0; i < numColours; i++)
+        {
+            colours.Add(new()
+            {
+                Red = ReadByte(),
+                Green = ReadByte(),
+                Blue = ReadByte()
+            });
+        }
+        return colours;
+    }
+
+    public List<TRColour4> ReadColour4s(long numColours)
+    {
+        List<TRColour4> colours = new((int)numColours);
+        for (long i = 0; i < numColours; i++)
+        {
+            colours.Add(new()
+            {
+                Red = ReadByte(),
+                Green = ReadByte(),
+                Blue = ReadByte(),
+                Unused = ReadByte()
```

Comment:
Is it really unused, or is it alpha? If it's unused maybe it's best to convert it to Colour3 and then write dummy values back?

Reply:
It's unused in the original TR2 and I'm fairly sure it's the same for TR3. But you are right, TR2Main treats it as the alpha channel, so I think renaming it is a good idea as it makes sense that this should be its purpose.

Reply:
Renamed now to `Alpha`. I've also fixed a previous hack where we were keeping track of imported colours using the `Unused` property. There is now a proper tracker class for 16-bit colour import, which will also quantize if the palette runs out of space - very unlikely in TR2/3, but it's good practice.